### PR TITLE
fix: removed duplicate video

### PR DESCRIPTION
**Pull request by jnikhila**
Removed the duplicate videos.

### CODE REVIEW

**Comment by spark-c** [comment]
@jnikhila  I can't find where the video is duplicated in the docs, was is it somewhere on that same JS Objects page?

File: website/docs/core-concepts/writing-code/javascript-editor-beta/README.md [hunk 1]
```diff
@@ -180,7 +180,6 @@ You’ll have to use the `debugger` keyword in the code snippet to pause the exe
 You must have the **browser’s developer console** **open** to see the debugger in action.
 :::
 
- <object data="https://www.youtube.com/embed/ZjleR7iM-xo?autoplay=0" width='750px' height='400px'></object>
```
Comment (jnikhila) [inline website/docs/core-concepts/writing-code/javascript-editor-beta/README.md:183]: @spark-c - here the video was duplicated. I'd removed one of them.

**Review by spark-c — APPROVED** [review]

**Comment by Pranay105** [comment]
@spark-c can you please the conflicts and merge this PR?

**Comment by jnikhila** [comment]
Resolved as part of PR #552.

**Pull request closed without merging**